projectile.el

```diff
@@ -2085,6 +2085,10 @@ For git projects `magit-status-internal' is used if available."
   "The command to use with `projectile-compile-project'.
 It takes precedence over the default command for the project type when set.
 Should be set via .dir-locals.el.")
+(defvar projectile-project-compilation-dir nil
```

Comment:
The other variables in that section aren't currently separated by blank lines. Should I add blank lines between them all?